modules/developer-edition/settings-page.php

```diff
@@ -42,7 +43,10 @@ function () {
 				null,
 				static::PAGE_ID
 			);
-			register_setting( Module::SETTINGS_KEY, 'elementor_dev_auto_update' );
+
+			register_setting( Module::SETTINGS_KEY, 'elementor_dev_auto_update', [
```

Comment:
Check it the DB is not updated with an option `elementor_dev_auto_update`

Reply:
I added "return false" at the end of the sanitize_callback method and it does not save it in the DB.